src/bulkOperationCommand.ts

```diff
@@ -40,6 +39,7 @@ type CreateJobOptions = {
 export abstract class BulkOperationCommand extends BulkBaseCommand {
   public static readonly baseFlags = {
     ...orgFlags,
+    ...BulkBaseCommand.baseFlags,
```

Comment:
I rewrote a bunch of plugin-data (should be simpler, only 1 place if at all). 

Can you do your change against https://github.com/salesforcecli/plugin-data/pull/843 probably [here]

(https://github.com/salesforcecli/plugin-data/pull/843/files#diff-ee69f30e4a3ba9ed75d920f9ff0387e1b64fe82e6f412bfb4c0ad078c792c78cR45)